tests/Storage/Microsoft.Bot.Builder.Azure.Tests.Common/Microsoft.Bot.Builder.Azure.Tests.Common.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What do you think about naming this one as Microsoft.BotBuilder.Tests.Common and move it up one level? We don't have anything Azure specific here so Azure in the name is misleading and like we discussed over the phone it can be used for MemoryStorage tests.

If we do go down this path, let's make sure we follow assembyName.Folder naming convention for namespaces and put the classes in this project inside a storage folder. 

We could also use consider using this common assembly as a place to dump some common helpers for BotBuilder tests.